packages/fxa-react/lib/test-utils/index.ts

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import path from 'path';
+import { readFileSync } from 'fs';
+import { FluentBundle, FluentResource, FluentVariable } from '@fluent/bundle';
+import { Pattern } from '@fluent/bundle/esm/ast';
+import { queries, Screen } from '@testing-library/react';
+
+type PackageName = 'settings' | 'payments' | null;
```

Comment:
I feel like this file is mostly there but is maybe a little bit messy, it was an iterative process getting it here... When I work on [FXA-6001](https://mozilla-hub.atlassian.net/browse/FXA-6001), I’m going to see if I can clean it up a little and see if it works nicer as a class. I went with `null` instead of `’test’` as an option here since I intended this to be a package name and `'test'` isn't a package but maybe I'll change that.

I’m wondering if we can/should load the `test/settings.ftl` bundle in Settings' `setupTests.ts` and make it globally available similar to the mock and then do the same for payments when we get there, since 99% of the time we’re going to want to load the English bundle to test against, and then the bundle loads once per full test run instead of per test file. 

I'm happy to make any changes here before merging but I do intend to look at all of this in that next ticket.